db/migrate/20201124145559_create_sdg_manager.rb

```diff
@@ -0,0 +1,7 @@
+class CreateSDGManager < ActiveRecord::Migration[5.2]
+  def change
+    create_table :sdg_managers do |t|
+      t.belongs_to :user, index: true, foreign_key: true
```

Comment:
I guess the index should be unique :thinking:, since we've added a uniqueness validation.